.github/workflows/publish.yml

```
@@ -0,0 +1,47 @@
+name: publish
+'on':
+  push:
+    branches:
+      ktp-docs-ci
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y pandoc git
+        pip3 install -r requirements-doc.txt
+        pip3 install scikit-learn-intelex
+        cd doc
+        source set_version.sh
+        echo Generating version $DOC_VERSION

```

```suggestion
        DOC_VERSION=$(python -m pip freeze | grep -oP "(?<=scikit-learn-intelex==)\d*\.\d*")
        echo Generating version $DOC_VERSION
        echo "DOC_VERSION=${DOC_VERSION}" >> "$GITHUB_ENV"
```
Add it to the github environment to save some work/lines going forward.